blazor/heatmap-chart/bubble-heatmap.md

```diff
@@ -9,6 +9,11 @@ documentation: ug
 
 # Bubble HeatMap in Blazor HeatMap Chart Component
 
+This section shows how to customize the appearance of the bubbles in the HeatMap component using the video below.
```

Comment:
This section shows how to customize the appearance of the HeatMap cells as bubbles. The below video demonstrates the same.